src/test/java/pages/BasePage.java

```diff
@@ -0,0 +1,138 @@
+package pages;
+
+import engine.config.reader.ConfigReader;
+import enums.CategoryMenuButton;
+import enums.FooterHyperLink;
+import enums.NavbarButton;
+import org.openqa.selenium.*;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import java.time.Duration;
+
+public class BasePage {
+    protected WebDriver driver;
+    protected WebDriverWait wait;
+    protected String baseUrl;
+
+    protected final String navbarXpath = "//div[@role='navigation']";
+    protected final String categoryMenuXpath = "//section[@id='categorymenu']";
+    protected final String footersocialXpath = "//section[@class='footersocial']";
+    protected final String footerlinksXpath = "//section[@class='footerlinks']";
+    public By searchInputLocator = By.xpath("//input[@id='filter_keyword']");
```

Comment:
Is there any reason behind not searching it directly with `By.id('filter_keyword')`?

Reply:
No. Changed to 
```java 
public By searchInputLocator = By.id("filter_keyword"); 
```